### PR TITLE
A quick copy edit pass

### PR DESCRIPTION
**Pull request by mairaw**
- Fixed some typos and punctuation
- Changed the voice to focus on the person (avoid we, our)
- Fixed some linting rules  
- Added some friendlier tone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
+
 # minimal.github.io
-tutorials and samples for minimal apis 
 
-- To run the site locally please go navigate to mini-docs and run - `npm run dev`
-- To add / update docs please go to src
-- To update / add menu items go to src/.vuepress/config.js
+Tutorials and samples for minimal APIs.
 
+- To run the site locally, navigate to mini-docs and run - `npm run dev`.
```
Comment (mairaw) [inline README.md:7]: where is this?

File: src/tutorial/crud.md [hunk 2]
```diff
@@ -144,26 +151,27 @@ The `Response body` will include the items just added.
...
- ![swagger-get-todos-item](https://user-images.githubusercontent.com/2546640/125182403-bd99e900-e1db-11eb-83bb-72eb89b4386f.gif)
 
-## Update an item 
+To check this out, you can go to `https://localhost:5001/todos/1` or use the swagger UI.
+![swagger-get-todos-item](https://user-images.githubusercontent.com/2546640/125182403-bd99e900-e1db-11eb-83bb-72eb89b4386f.gif)
+
+## Update an item
 
-To update an existing item add the code below `GET /todos/{id}` route we created above.
+To update an existing item add the following code under the `GET /todos/{id}` route you created earlier.
```
Comment (mairaw) [inline src/tutorial/crud.md:166]: I wasn't sure if this change was correct here, so please double check

File: src/tutorial/databases.md [hunk 3]
```diff
@@ -2,29 +2,31 @@
...
-## Setup SQLite database
-**Setup SQLite Database using Entity Framework Core(EF Core)***
+**Set up SQLite Database using Entity Framework Core(EF Core)**
 
 **Install the following tools and packages**
 
 Using .NET CLI / Visual Studio package manager UI, install the following packages:
 
-**[SQLite EF Core Database Provider](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Sqlite/6.0.0-preview.5.21301.9)** : can access many different databases through plug-in libraries called [database providers](https://docs.microsoft.com/en-us/ef/core/providers/?tabs=dotnet-core-cli). The package below is the SQLite database provider for EF Core.
+**[SQLite EF Core Database Provider](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Sqlite/6.0.0-preview.5.21301.9)**: can access many different databases through plug-in libraries called [database providers](https://docs.microsoft.com/ef/core/providers/?tabs=dotnet-core-cli). The following package is the SQLite database provider for EF Core.
```
Comment (mairaw) [inline src/tutorial/databases.md:23]: we're now in preview 7. should url and commands be updated?

File: README.md (hunk 1 above)
Comment (LadyNaggaga) [reply README.md:7]: Will update the instructions it should be  minimal.github.io

File: src/tutorial/databases.md (hunk 3 above)
Comment (LadyNaggaga) [reply src/tutorial/databases.md:23]: This should be RC1 for the next release .

**Pull request merged**